site/content/installation/installing-ngf/manifests.md

```diff
@@ -138,6 +138,16 @@ kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric
 
 {{% /tab %}}
 
+{{%tab name="NGINX in debug mode"%}}
+
+Deploys NGINX Gateway Fabric with NGINX OSS in debug mode.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.4.0/deploy/nginx-debug/deploy.yaml
```

Comment:
@sjberman do you think this link should point to the next release? Or just keep it as is and we will update it during the release?

```suggestion
kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.5.0/deploy/nginx-debug/deploy.yaml
```

Reply:
we usually use the current release and update it automatically at release time